inst/templates/custom_check_template

```diff
@@ -0,0 +1,51 @@
+{{name}} <- function(tbl, file_path{{#extra_args}}, extra_arg = NULL{{/extra_args}}) {
```

Comment:
If I understand the custom function mechanism correctly, we will need to include all the arguments for `validate_model_data` (see https://hubverse-org.github.io/hubValidations/articles/custom-functions.html#default-configuration) so that we don't get `unused argument` errors

Reply:
Not quite. All the objects listed in that vignette are available in the caller function and can be passed **if required by any custom function**.

Reply:
I'm going to pull that section of the article into a child doc we can repeat in both and will try to clarify that point

Reply:
Thank you for the clarification! I will admit that I didn't look too deeply at the code evaluating the custom functions and I am impressed!